epub33/core/index.html

```diff
@@ -1292,7 +1292,14 @@ <h5>The <code>package</code> Element</h5>
 
 						<p id="attrdef-package-version">The <code>version</code> attribute specifies the EPUB
 							specification version to which the given EPUB Package conforms. The attribute MUST have the
-							value "<code>3.0</code>" to indicate compliance with this version of the specification.</p>
+							value "<code>3.0</code>" to indicate conformance with EPUB 3.</p>
+
+						<div class="note">
+							<p>Updates to this specification do not represent new versions of EPUB 3 (i.e., each new 3.X
+								specification is a continuation of the EPUB 3 format). The working group is committed to
+								ensuring that changes do not invalidate existing content, allowing the
```

Comment:
in practice, this is not strictly true. A few changes, sometimes to specifications not under the WG's control (like HTML), are not backward compatible. We had a few cases in 3.2 and it will likely happen again.
Could we maybe loosen this statement a bit?

Reply:
Being committed to something doesn't mean it can't happen. That's why I avoided strong wording that would suggest changes can't happen. Whether other specifications in other groups might change is getting a bit too deep into the weeds for a note.

But if the intention is not apparent enough, the latter sentence could maybe be modified to:

"The working group is committed to minimizing any changes that would invalidate existing content..."

Reply:
FWIW, I prefer this latter wording 😊